src/overlays/actors/ovl_Bg_Spdweb/z_bg_spdweb.c

```diff
@@ -350,23 +350,20 @@ void func_809CE4C8(BgSpdweb* this, PlayState* play) {
     CollisionCheck_SetAC(play, &play->colChkCtx, &this->collider.base);
 }
 
-#ifdef NON_MATCHING
 void func_809CE830(BgSpdweb* this, PlayState* play) {
```

Comment:
Think after all the work you did on understanding it you might as well sketch out some docs on this function.